README.md

```diff
@@ -32,7 +32,7 @@ Make sure you replace the paths in the following instructions with your own k4a
 It is important to replace `1.2.0` with your installed version of the SDK.
 
 ```
-pip install pyk4a --global-option=build_ext --global-option="-IC:\Program Files\Azure Kinect SDK v1.2.0\sdk\include" --global-option="-LC:\Program Files\Azure Kinect SDK v1.2.0\sdk\windows-desktop\amd64\release\lib"
+pip install pyk4a --no-use-pep517 --global-option=build_ext --global-option="-IC:\Program Files\Azure Kinect SDK v1.4.1\sdk\include" --global-option="-LC:\Program Files\Azure Kinect SDK v1.4.1\sdk\windows-desktop\amd64\release\lib"
```

Comment:
Like as this option helps: `--no-use-pep517`
There some glues: 
* https://github.com/pypa/pip/issues/6398
* https://github.com/pypa/pip/issues/5771
* https://github.com/pypa/pip/issues/4383